### ISSUE TITLE
Create a "Contributors" guide on BackdropCMS.org

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#486]
**Problem**

For new users it can be difficult and confusing to figure out how to contribute to BackdropCMS. For many people there is an intimidation factor and fear of doing things wrong that prevents them from contributing a PR, commenting on an issue, or reviewing/testing a PR. The more detailed and explicit we can be about how to contribute, the more likely people are to become active in the community. 

**Proposed Solution**

Create a "contributors guide" on BackdropCMS org similar to the "Getting Involved Guide" on Drupal.org (https://www.drupal.org/getting-involved-guide)

I recommend focusing on the basics of how the community works and something akin to the contributor tasks sections on D.O. https://www.drupal.org/contributor-tasks

I am willing to take some initiative on helping organize and write these guides. I would ask for input and feedback on tasks that I don't understand well. 

But, I'd like some feedback on the idea before I get started. Also, any historical information about whether or not there has been discussion of this before.

**Comment by stpaultim** [comment]
Agreed. This contributor guide would not be about code. It's about helping people contribute whatever their skill level. I just looked at how existing documentation is structured on BackdropCMS.org and I'm seeing two main sections.

1. Backdrop CMS Handbook
1.1 [About Backdrop CMS](https://backdropcms.org/about)
1.2 Backdrop Contributors Guide (PROPOSED)
1.3 [Backdrop Developer notes](https://backdropcms.org/developers)
2. [Backdrop CMS User Guide](https://backdropcms.org/user-guide)

I wondered if the Contributor guides should be a part of the CMS Handbook or if it should be a section on it's own. I'm inclined to start with it as a section of the CMS Handbook (see above). 

@klonos also just raised the question of whether or not there should be a listing for Contributors on the [resources](https://backdropcms.org/support/resources) page. 

My plan, pending any feedback to the contrary, it to work up a couple of pages in DRAFT mode for now and then get some feedback before publishing anything. Does this make sense?

**Comment by jenlampton** [comment]
Sounds great!

**Comment by klonos** [comment]
Yes, please go ahead and create those drafts @stpaultim. Please ping me on Gitter when you need a review.

**Comment by stpaultim** [comment]
I went ahead and started a contributors guide. It may come in handy for OSD (Open Source Day 2020) and/or it might be a great chance to get help drafting some additional content. 

Any OSD participants, please feel free to comment on what should be included or jump in and help us write some copy.

**Comment by jenlampton** [comment]
One note that anything high-level can go on the main site, but as soon as we start talking about code or PRs that needs to be a page on the API site. (We should feel free to cross-link between them as much as necessary, though!)
